src/components/DatePickerRange.react.js

```diff
@@ -132,6 +132,11 @@ export default class DatePickerRange extends Component {
 
   render() {
     const { setProps, fireEvent } = this.props;
+    let verticalFlag = true;
+    if (this.props.calendar_orientation === 'vertical') {
```

Comment:
put `calendar_orientation` in the `this.props` assignment above: `const { calendar_orientation, setProps, fireEvent } = this.props;`